### DISCUSSION TITLE
Overflow of log (or other transforms)

### DISCUSSION THREAD

**Discussion by PaulWAyers** [theochem/AtomDB#1] [General]
@gabrielasd and @msricher do we have something to control for the fact that the log of the atomic density/orbital energy/orbital densities may fail when the argument of the log is too close to zero? Something like: 
    https://github.com/theochem/denspart/blob/e21de342078035669832b693cb49b269870a6e02/denspart/vh.py#L434

**Comment by gabrielasd** [comment]
That's a good point, and no, we don't have it at the moment. We should add it to this class
https://github.com/theochem/AtomDB/blob/a4e8f57a502adad163cb07e7beb3b35f7dc27ca6/atomdb/api.py#L382

**Comment by PaulWAyers** [comment]
I've made this into an issue. Thanks @gabrielasd
